tests/acceptance/features/coreApiShareOperationsToShares2/uploadToShare.feature

```diff
@@ -165,14 +165,14 @@ Feature: sharing
   @skipOnGraph
```

Comment:
Could you check these tags?
```suggestion
```

Reply:
The PR related to uploading file on share is failing and thus even if the quota code is updated to graph, this doesn't work.

Reply:
then we have to fix it or remove the test scenario if it's not valid.